recipes/pyknos/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyknos" %}
+{% set version = "0.16.0" %}
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyknos-{{ version }}.tar.gz
+  sha256: 4e1db834d8a5fd847882a081937732fea6798668b72293ae052765e7bfc371c3
+
+build:
+  noarch: python
```

Comment:
Windows builds will fail here as expected, given that PyTorch doesn't support Windows. The rest is `noarch` though so can keep it as `noarch: python`.